src/m365/spo/commands/hubsite/hubsite-get.ts

```diff
@@ -25,10 +28,36 @@ class SpoHubSiteGetCommand extends SpoCommand {
     return 'Gets information about the specified hub site';
   }
 
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.id = (!(!args.options.id)).toString();
+    telemetryProps.includeAssociatedSites = args.options.includeAssociatedSites === true;
+    return telemetryProps;
+  }
+
+  private async getAssociatedSites(spoAdminUrl: string, logger: Logger, args: CommandArgs): Promise<AssociatedSite[]> {
+    const options: SpoListItemListCommandOptions = {
+      output: 'json',
+      debug: args.options.debug,
+      verbose: args.options.verbose,
+      listTitle: 'DO_NOT_DELETE_SPLIST_TENANTADMIN_AGGREGATED_SITECOLLECTIONS',
+      webUrl: spoAdminUrl,
+      filter: `HubSiteId eq '${args.options.id}'`,
+      fields: 'Title,SiteUrl,SiteId'
+    };
+
+    const output = await Cli
+      .executeCommandWithOutput(SpoListItemListCommand as Command, { options: { ...options, _: [] } });
+
+    return JSON.parse(output.stdout) as AssociatedSite[];
```

Comment:
lets change the output of this method to `Promise<CommandOutput>`

and lets return just 
```
return Cli.executeCommandWithOutput(SpoListItemListCommand as Command, { options: { ...options, _: [] } });
```